src/com/osreboot/ridhvl2/HvlMath.java

```diff
@@ -18,6 +20,30 @@ private HvlMath(){}
 	 */
 	public static final float PI = (float)Math.PI;
 
+	/**
+	 * A {@linkplain Random} object used for random number generation.
+	 */
+	private static Random seed = new Random();
+
+	/**
+	 * Returns a random <code>int</code> falling between <code>minArg</code> and
+	 * <code>maxArg</code>, inclusive.
+	 * 
+	 * @param minArg the smallest possible <code>int</code> value that can be returned
+	 * @param maxArg the largest possible <code>int</code> value that can be returned
+	 * @return A random <code>int</code> between these two points
+	 */

```

Comment:
Please indicate which of minArg and maxArg are inclusive/exclusive in the comment.